apps/src/blockly/addons/cdoUtils.js

```diff
@@ -14,6 +14,10 @@ export function setHSV(block, h, s, v) {
   block.setColour(Blockly.utils.colour.hsvToHex(h, s, v * 255));
 }
 
+export function injectCss() {
```

Comment:
Not necessary to update for this PR, but I wonder if we should separate out things that are true utils (eg, `getBlockFields`) from these situations that I think would be better described as "shims" (ie, we have method signatures that are different between CDO and Google blockly for the same function, and need to do some wrapping/etc to make them align).

Reply:
We can use a shim for `svgResize` as well.

Reply:
Discussed on Slack and we'll keep helper functions in this PR in wrappers but plan to do some reorganization of the wrappers soon.